meteor/sofie-server/data-access/repositories/cache/cached-rundown-repository.ts

```diff
@@ -35,4 +36,12 @@ export class CachedRundownRepository implements RundownRepository {
 			this.cachedRundowns.set(rundown.id, rundown)
 		}
 	}
+
+	public async deleteRundown(rundownId: string): Promise<void> {
+		const cacheDeletionResult: boolean = this.cachedRundowns.delete(rundownId)
```

Comment:
You could just call this `wasDeleted`. That implies that it is a boolean. `CacheDeletionResult` implies that it is an object

Reply:
Renamed to `wasDeleted`